api/handlers/job.go

```diff
@@ -135,19 +158,19 @@ func (h *Job) handleOrgDelete(ctx context.Context, resourceGUID, jobGUID string)
 			jobGUID,
 			[]presenter.JobResponseError{},
 			presenter.StateProcessing,
-			orgDeletePrefix,
+			jobType,
 			h.serverURL,
 		), nil
 	} else {
 		return presenter.ForJob(
 			jobGUID,
 			[]presenter.JobResponseError{{
 				Code:   10008,
-				Detail: fmt.Sprintf("CFOrg deletion timed out. Check for lingering resources in the %q namespace", org.GUID),
+				Detail: fmt.Sprintf("%s deletion timed out. Check for lingering resources in the %q namespace", resourceType, resourceGUID),
```

Comment:
This is a nitpick, but wanted to put it out for consideration. I'm wondering if there is a simpler word than "lingering" that we could use in this message.

```suggestion
				Detail: fmt.Sprintf("%s deletion timed out. Check for resources remaining in the %q namespace", resourceType, resourceGUID),
```